platform/src/main/java/org/hillview/table/api/ISampledRowIterator.java

```diff
@@ -0,0 +1,13 @@
+package org.hillview.table.api;
+
+/**
+ * An iterator over the rows that were sampled with some rate
```

Comment:
this comment is incomplete; it was sampled from a MembershipSet.
The rate represents the sampling rate from the parent MembershipSet.